policy/modules/services/postfix.te

```diff
@@ -752,6 +760,10 @@ optional_policy(`
 	systemd_use_nss(postfix_showq_t)
 ')
 
+optional_policy(`
+	unconfined_run_to(postfix_showq_t, postfix_showq_exec_t)
```

Comment:
This interface is meant for downstream use only, so this should change to a call in unconfined.te to a postfix interface.

Reply:
I removed that, I'll add a Postfix interface in a future PR.